site/block-diagram/block_diagram.css

```diff
@@ -0,0 +1,847 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * This is a CSS library for building block diagrams out of simple semantic HTML.
+ *
+ * We're not CSS engineers, so I've done my best not to include hacks, unstable
+ * features, or massive complexity. Remaining complexity will be documented
+ * or wrapped into variables to make intent clear.
```

Comment:
After reading this file, I think you can safely call yourself a CSS engineer.

Reply:
Oh I hope not